### PR TITLE
Flag admin1 capitals

### PR DESCRIPTION
**Pull request by stepps00**
Fixes a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/58

This PR:

- Adds `wof:capital` to admin1 records, equal to the `wof:id` of the admin1's capital locality
- Adds `wof:capital_of` to locality records, equal to the `wof:id` of the parent admin1
- Validates changes

Can be merged once approved.

### CODE REVIEW

**Review by nvkelso — APPROVED** [review]

File: data/112/602/497/5/1126024975.geojson [hunk 1]
```diff
@@ -239,10 +239,11 @@
     "wof:belongsto":[
         102191569,
         85632573,
-        85667969,
-        1376833607
+        1376833607,
+        85667969
     ],
     "wof:breaches":[],
+    "wof:capital_of":85667969,
```
Comment (nvkelso) [inline data/112/602/497/5/1126024975.geojson:246]: This one also needs to be a list?

**Review by nvkelso — CHANGES_REQUESTED** [review]

File: data/890/455/647/890455647.geojson [hunk 2]
```diff
@@ -392,10 +392,11 @@
     "wof:belongsto":[
         102191569,
         85632573,
-        85667951,
-        1376833605
+        1376833605,
+        85667951
     ],
     "wof:breaches":[],
+    "wof:capital_of":85667951,
```
Comment (nvkelso) [inline data/890/455/647/890455647.geojson:399]: Another example of needing to be in a list.

Comment (stepps00) [reply data/890/455/647/890455647.geojson:399]: Fixed in 8d898b1.

Comment (stepps00) [reply data/890/455/647/890455647.geojson]: This was fixed in 56a8ef4.

File: data/112/602/497/5/1126024975.geojson (hunk 1 above)
Comment (stepps00) [reply data/112/602/497/5/1126024975.geojson]: fixed in most recent commits.

**Comment by stepps00** [comment]
@nvkelso this PR should be ready to ship. If it looks alright, I'll merge the other per-country PRs with similar changes.

**Pull request merged**